### PR TITLE
Require support tickets to be *initiated* by community reps

### PR DESCRIPTION
**Pull request by yuvipanda**
I'm working with Sarah in systematizing how we would respond to a request like https://2i2c.freshdesk.com/a/tickets/1259. Among many issues, one is that we don't actually know who the person asking for this is or if they are authorized to ask for this change.

See https://github.com/2i2c-org/infrastructure/issues/3048#issuecomment-1907195217 for the process of how the linked airtable came to be.

----
📚 Documentation preview 📚: https://2i2c-team-compass--797.org.readthedocs.build/en/797/

### CODE REVIEW

**Comment by yuvipanda** [comment]
There may be updates to what airtable exactly is going to be used, see conversation on linked ticket.

File: projects/managed-hubs/support.md [hunk 1]
```diff
@@ -179,19 +179,38 @@ The current iteration of the workflow states each step and who should be respons
 
 When a new ticket lands in Freshdesk under the support group and it is not an incident, we aim to respond within 24 working hours with a suggested next action. The next steps should be followed when resolving a ticket:
 
-1. `Who: Support steward`
+1. `Who:Support steward`
 
-   **First 24h initial ticket evaluation**. In the first 24h a support ticket was opened, you should do an initial evaluation of the ticket and ask the {term}`Community Representative` about any additional information you may need.
+   First, we determine if the person *initiating* the support ticket is *authorized* to do actually do so. While we may interact with many folks
```
Comment (sgibson91) [inline projects/managed-hubs/support.md:184]:
```suggestion
   First, we determine if the person *initiating* the support ticket is *authorized* to actually do so. While we may interact with many folks
```

File: projects/managed-hubs/support.md [hunk 2]
```diff
@@ -179,19 +179,38 @@ The current iteration of the workflow states each step and who should be respons
 
 When a new ticket lands in Freshdesk under the support group and it is not an incident, we aim to respond within 24 working hours with a suggested next action. The next steps should be followed when resolving a ticket:
 
-1. `Who: Support steward`
+1. `Who:Support steward`
```
Comment (consideRatio) [inline projects/managed-hubs/support.md:182]:
```suggestion
1. `Who: Support steward`
```

File: projects/managed-hubs/support.md [hunk 3]
```diff
@@ -179,19 +179,38 @@ The current iteration of the workflow states each step and who should be respons
...
 
-   **First 24h initial ticket evaluation**. In the first 24h a support ticket was opened, you should do an initial evaluation of the ticket and ask the {term}`Community Representative` about any additional information you may need.
+   First, we determine if the person *initiating* the support ticket is *authorized* to do actually do so. While we may interact with many folks
+   from a community during resolution of a ticket, we constrain who can *initiate* a ticket to {term}`Community Representative`s only. This prevents our
+   support staff from being overwhelmed by tickets that need to be handled elsewhere. If the person *initiating* the ticket is not a community
+   representative, the support steward should cc the community representatives, and ask for approval. The support steward *may* choose to use the following
+   email template:
```
Comment (sgibson91) [inline projects/managed-hubs/support.md:188]:
```suggestion
   email template also available as a [Canned Response](https://2i2c.freshdesk.com/a/admin/canned_responses/folders):
```

I added this into our canned responses on Freshdesk as well so it's really easy to find.

File: projects/managed-hubs/support.md [hunk 4]
```diff
@@ -179,19 +179,38 @@ The current iteration of the workflow states each step and who should be respons
...
+
+   > Hello <names of community representatives>,
+   >
+   > I'm cc'ing you on this support ticket we received from a member of your community. To streamline our support process, 2i2c is accepting
+   > support requests only from communtity representatives. Can you read through the request, and let us know how you wish to proceed?
```
Comment (sgibson91) [inline projects/managed-hubs/support.md:193]:
```suggestion
   > support requests only from community representatives. Can you read through the request, and let us know how you wish to proceed?
```

**Review by sgibson91 — APPROVED** [review]

File: projects/managed-hubs/support.md [hunk 5]
```diff
@@ -200,39 +219,39 @@ When a new ticket lands in Freshdesk under the support group and it is not an in
...
 
    **Revisit the impact metadata**. Once a week (at minimum) the [support view in the **Eng & Prod** board](https://github.com/orgs/2i2c-org/projects/22/views/47) should be revisited to validate the impact level on support-related issues. Currently, we allocate a 30-minute working session every Wednesday (open to everyone to participate) to perform such impact revision and further prioritization ("planned" reactive) every other week (see step 7 for more details).
-     
-5. `Who: Support steward`
 
-   **Add a reference/link to the created engineering issue inside the Freshdesk ticket**. You can use an internal note or make it public when you communicate back to the Community Representative in step 6. Also, move the status of the ticket to the "Pending" state.
-   
 6. `Who: Support steward`
 
+   **Add a reference/link to the created engineering issue inside the Freshdesk ticket**. You can use an internal note or make it public when you communicate back to the Community Representative in step 6. Also, move the status of the ticket to the "Pending" state.
```
Comment (consideRatio) [inline projects/managed-hubs/support.md:251]:
```suggestion
   **Add a reference/link to the created engineering issue inside the Freshdesk ticket and set ticket status to Pending**. You can use a private note or make it public when you communicate back to the Community Representative in the next step.
```

**Review by consideRatio — APPROVED** [review]
I think we should also try to label those that are authorized in freshdesk by editing the contact details in freshdesk.

There are these things we can configure in a 2i2c freshdesk contact:
- Full name
- Title
- Email
- Work phone
- Twitter
- External ID
- Address
- Company
- Tags
- About

Looking at this and testing what is visible in what ways, I conclude tags wasn't helpful.

- I suggest we either append this kind of status to the "Full name" for maximum visibility, or add it in the "Title" where its seen below the name in the user profile.
- I suggest we also make sure that Company is populated correctly

Check for example Victor's Samuel user profile, also click on company and note that we can use this to navigate to other members of the same community - but perhaps not all if we don't work to organize them.

With a procedure like this in place to update freshdesk users, we can use that information first and formost and fall back to airtable only when needed.

**Comment by yuvipanda** [comment]
This is now pending https://github.com/2i2c-org/infrastructure/issues/3048#issuecomment-1909011115.

@consideRatio I agree freshdesk would be a good end location for that. However, given that currently partnerships is using airtable as the *single source of truth*, I think that's the place the 'source of truth' should be. Ideally, we'd be able to do some sort of 'sync' between airtable and freshdesk so freshdesk can show what's in airtable. But I think just having a single page where you can ctrl-f is already a big improvement over what we have now.

File: projects/managed-hubs/support.md (hunk 5 above)
Comment (yuvipanda) [reply projects/managed-hubs/support.md:251]: although this made me realize that I'm going to have to touch this anyway, as the step names have changed. So I'll just incorporate your suggestion into it too @consideRatio when i edit it!

**Review by damianavila — APPROVED** [review]

**Review by GeorgianaElena — APPROVED** [review]
:heart:

**Comment by GeorgianaElena** [comment]
Thank you @yuvipanda 🚀

**Comment by yuvipanda** [comment]
w00t, thank you so much @GeorgianaElena

**Pull request merged**